tests/rptest/tests/resource_limits_test.py

```diff
@@ -88,16 +88,18 @@ def test_cpu_limited(self):
 
         rpk = RpkTool(self.redpanda)
 
-        # Three nodes, each with 1 core, 7000 partition-replicas
-        # per core, so with replicas=3, 7000 partitions should be the limit
+        # Three nodes, each with 1 core, 1000 partition-replicas
```

Comment:
Not necessarily this test, but it'd be great to add an upgrade test that validates that this config doesn't change over jumping from 23.1.1.